chainio/clients/builder.go

```diff
@@ -16,12 +15,12 @@ import (
 )
 
 type BuildAllConfig struct {
-	EthHttpUrl                    string `yaml:"eth_http_url"`
-	EthWsUrl                      string `yaml:"eth_ws_url"`
-	BlsRegistryCoordinatorAddr    string `yaml:"bls_registry_coordinator_address"`
-	BlsOperatorStateRetrieverAddr string `yaml:"bls_operator_state_retriever_address"`
-	AvsName                       string `yaml:"avs_name"`
-	PromMetricsIpPortAddress      string `yaml:"prometheus_metrics_ip_port_address"`
+	EthHttpUrl                 string `yaml:"eth_http_url"`
+	EthWsUrl                   string `yaml:"eth_ws_url"`
+	RegistryCoordinatorAddr    string `yaml:"bls_registry_coordinator_address"`
```

Comment:
does the `yaml` name needs to change too? remove `bls_` ?

Reply:
[ab1bd6a](https://github.com/Layr-Labs/eigensdk-go/pull/91/commits/ab1bd6aa67d462eadff91c8b776b3a632e9fcd72)
Just removed all the yaml tags actually. Not sure why we even had them, don't see why we would ever build this from a config file. Let me know if you think otherwise.